app/timetables/tests/test_models.py

```diff
@@ -235,3 +237,49 @@ def test_zero_cycle_day_value_cannot_be_saved(self):
             timetable=self.timetable_object)
 
         self.assertRaises(ValidationError, menu_item_three.save)
+
+
+class EventTest(TestCase):
```

Comment:
@andela-snwuguru after my discussion with @andela-ooshodi, It is right we test creation of models. We should also add that one can update and delete to ensure that we get the right response when we delete a related model. So yes, go ahead with testing all that for this ticket. I will make a new ticket for updating other tests.